sdk/Pulumi/Core/Input.cs

```diff
@@ -51,25 +51,33 @@ public static class InputExtensions
         /// <summary>
         /// <see cref="Output{T}.Apply{U}(Func{T, Output{U}?})"/> for more details.
         /// </summary>
+#pragma warning disable CA1715 // Identifiers should have correct prefix
```

Comment:
Not convinced these identifier suffix/prefix rules are worth having turned on.

Reply:
I was initially on the fence on these too, but I kind of want them on. I want to make sure we're naming Attributes and Exceptions with the right suffix (which is something I've seen previously in code reviews, that I luckily happened to catch).

I could be convinced to disable the prefix rule, since we tend to use `U` for generic type names rather than the preferred "prefix with `T`". But it does check that interfaces are prefixed with `I`, which I kind of want to keep. I wish I could easily ignore the generic type name one but keep the interface rule.

Reply:
Maybe we should just do a pass to change to T prefixes then, that seems to be the bulk of the warning disables.